hal/armv7a/zynq7000/peripherals.h

```diff
@@ -37,12 +37,6 @@
 #define UART0_CLK amba_uart0_clk
 #define UART1_CLK amba_uart1_clk
```

Comment:
It would be great to remove file `peripherals.h` in the future and keep all hardware settings in `board_config.h`. My suggestion is to move some of the defines to source files and board_config.h